.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: ruby
 
+cache: bundler
```

Comment:
this shortens build times tremendously, and will still pull in latest gems allowed because we dont keep gemfile.lock in repo

you can see in the bundle install outputs `Using <gemname> <version>` rather than `Installing <gemname> <version>` on each build now, and chefspec is down to about 90 seconds rather than 5 min like it was previously.
more info https://docs.travis-ci.com/user/caching/#Caching-directories-(Bundler%2C-dependencies)